sdk/include/ds/xoroshiro.h

```diff
@@ -85,7 +142,15 @@ namespace ds::xoroshiro
 	 *
 	 * Parameters from https://prng.di.unimi.it/xoroshiro128plus.c .
 	 */
-	using P128R64 = detail::XorOshiro<uint64_t, uint64_t, 24, 16, 37>;
+	using P128R64 = detail::XorOshiro<uint64_t,
+	                                  uint64_t,
+	                                  24,
+	                                  16,
+	                                  37,
+	                                  0xdf900294d8f554a5,
+	                                  0x170865df4b3201fc,
+	                                  0xd2a98b26625eee7b,
+	                                  0xdddf9b1090aa7ac1>;
```

Comment:
"Let's see... types, small numbers... omg".  I trust that those are the right big numbers.

Reply:
Updated the comment to provide a citation for the magic numbers.  I am 100% confident that they are, in fact, numbers.